.github/workflows/build-test-image.yml

```diff
@@ -0,0 +1,44 @@
+name: Build Test Image
+
+on:
+  workflow_run:
+    workflows: [Build/Test]
+    types: [completed]
```

Comment:
I don't think you need this. this needs to instead be replaced by a merge group trigger